EmpireLegends/BiddingFacility.cpp

```diff
@@ -25,32 +25,48 @@ int BiddingFacility::getPlayerBid(Player* player)
 	return bids[player];
 }
 
-void BiddingFacility::placeBids(vector<Player*> biddingPlayers) {
+ string BiddingFacility::placeBids(vector<Player*> biddingPlayers) {
 
 	auto playerBid = 0;
 	string temp;
 
 	// each players places their bid
-	for (auto& player : biddingPlayers)
-	{
-		do {
-			std::cout << "\n" << player->getName() << " place your bid: ";
-			std::cin >> playerBid;
-		} while (playerBid > player->getCoins() || playerBid < 0);
-
-		system("cls");
+	
+		for (auto& player : biddingPlayers)
```

Comment:
just a question of formatting, but this whole section has an extra tab in front of it. you can simply select this whole section then hit `shift + tab` and it'll move it all one tab back 👍